blocks/article-copyright/article-copyright.js

```diff
@@ -0,0 +1,21 @@
+import { getMetadata } from '../../scripts/aem.js';
+import { getLocaleInfo } from '../../scripts/scripts.js';
+
+export default async function decorate(block) {
+  const copyrightPath = getMetadata('copyright-path');
```

Comment:
to keep it simple and follow YAGNI, I would go with having this path hardcoded rather than reading metadata.
if the authors want to see how a new copyright message looks like, they will simply preview first the changes and see them in `.page`.
at least at the moment, I don't see when would we need to have it configurable per article.